.github/workflows/image.yaml

```diff
@@ -0,0 +1,56 @@
+name: Image
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: /home/runner/work/argo-cd/argo-cd
+    steps:
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.12.1'
```

Comment:
Hm, should this rather be `1.13.6`, as used throughout the build system?

Reply:
nice catch. I forgot about it. Fixed.